app/controllers/api/v1/assessments_controller.rb

```diff
@@ -29,8 +29,10 @@ def index
       render api_response(success: false, message: 'must include "q" query parameter')
       return
     end
-    found = A15K::Metadata.api.query(params[:q])
-    render api_response data: Assessment.where(id: found.map(&:id))
+    found = A15K::Metadata.api.query(params[:q]).index_by(&:id)
+    assessments = Assessment.where(id: found.keys)
+    assessments.each{|a| a.metadata = { tags: ['one', 'great', 'day' ] } }
```

Comment:
are you intending to leave this in here as an example?

Reply:
Yah, that's a stub until https://github.com/a15k/metadata-repository/pull/7 is ready